client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -396,6 +404,67 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
         };
     }
 
+    /**
+     * Function to get a list with all favorite motions.
+     *
+     * @returns An array containing all favorite motions.
+     */
+    public motionsAsFavorite(): ViewMotion[] {
+        return this.motionRepo.getSortedViewModelList().filter(motion => motion.star);
+    }
+
+    /**
+     * Function to get a list of contained motions in a category.
+     *
+     * @param category The category
+     *
+     * @returns An array with all the motions in the given category.
+     */
+    public motionsInCategory(category: Category): ViewMotion[] {
```

Comment:
The same argument as for favorite motions: Please create an issue for this and assign me - I'll change the categories a bit, if we do the sub-categories. With these changes, this query can be simplified later.